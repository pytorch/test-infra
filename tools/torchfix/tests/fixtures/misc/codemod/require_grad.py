import torch
x = torch.zeros(1)
x.require_grad = False
x.require_grad = True
