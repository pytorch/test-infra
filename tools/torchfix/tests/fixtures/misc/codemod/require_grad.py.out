import torch
x = torch.zeros(1)
x.requires_grad = False
x.requires_grad = True
